controllers/servers/host_definer/storage_manager/host_definer_server.py

```diff
@@ -61,9 +66,62 @@ def _get_host_name(self, initiators, array_mediator):
         logger.debug("host found : {}".format(found_host_name))
         return found_host_name
 
+    def _update_host_ports(self, request, host, array_mediator):
```

Comment:
(comment for most new private methods)
can we move it somewhere else? (i.e. utils, helper...)

Reply:
why?

Reply:
we can do a brainstorming on which functions to import out to different classes if you want, as we did with the Kubernetes manager in the hostDefiner

Reply:
just not to overload the class.. can you open a ticket for a discussion?

Reply:
ok

Reply:
CSI-4867